data/mods/Magiclysm/Spells/magus.json

```diff
@@ -1,4 +1,28 @@
 [
+  {
+    "id": "shadow_field",
+    "type": "SPELL",
+    "name": "Shadow Field",
+    "description": "Cast a field of intense shadows.",
+    "effect": "attack",
+    "field_id": "fd_darkness",
+    "min_field_intensity": 1,
+    "max_field_intensity": 1,
+    "shape": "blast",
+    "valid_targets": [ "self", "ground" ],
+    "min_duration": 6000,
+    "max_duration": 90000,
+    "min_range": 10,
+    "max_range": 10,
+    "duration_increment": 2800,
+    "max_level": 30,
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_energy_cost": 250,
+    "base_casting_time": 500,
+    "difficulty": 1,
+    "flags": [ "NO_LEGS", "SOMATIC", "CONCENTRATE" ]
```

Comment:
i noticed that this spell has no area of effect. additionally, its casting time is a little high for combat magic at 5 seconds.

Reply:
Is boosting the energy cost (or difficulty) needed to compensate for a casting time decrease?

Reply:
AOE could be gotten from Ethereal Grasp, although perhaps slightly better (Ethereal Grasp is Classless). Also note that if the radius gets to 11+ (10+?), the range will need extending so it isn't automatic for the caster to be caught in it.

Reply:
If the casting time is 500 ticks = 5 seconds, agreed; I'm open to suggestions on balance on that score.

Also, my initial suggestion on that score is an AOE of 1, which should offer a 3x3 field. Or would 1-2 be more in-line?

Reply:
Ethereal Grasp is Classless (and thus supposed to be worse than a classed spell of the same tier), and has one of 2-20 (increment 1). It may be distinctly shorter-lasting, however (see below).

Reply:
> Is boosting the energy cost (or difficulty) needed to compensate for a casting time decrease?

it's hard to say since it didn't have an AOE.

Reply:
> If the casting time is 500 ticks = 5 seconds, agreed; I'm open to suggestions on balance on that score.
> 
> Also, my initial suggestion on that score is an AOE of 1, which should offer a 3x3 field. Or would 1-2 be more in-line?

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/mods/Magiclysm/magic_balance.md#casting-time
1-4 is probably good imo

Reply:
> 1-4 is probably good imo for aoe

With a 1-4 AOE - maybe casting time 200 with no other changes, 100 with balance changes?